libs/components/code-examples/src/lib/modules/forms/input-box/custom-form-errors/example.component.html

```diff
@@ -0,0 +1,23 @@
+<form [formGroup]="formGroup">
```

Comment:
Adding an example that demonstrates a custom form error in isolation, so that it's easier for consumers to find.